jsUnit.js

```diff
@@ -0,0 +1,28 @@
+/* global process require */
```

Comment:
Let's move this file to `SETUP/tests` too so it isn't hanging around in deployed site code.